deepchecks/nlp/datasets/classification/just_dance_comment_analysis.py

```diff
@@ -75,27 +185,28 @@ def load_data(data_format: str = 'TextData', as_train_test: bool = True, use_ful
     if data_format.lower() not in ['textdata', 'dataframe']:
         raise ValueError('data_format must be either "Dataset" or "Dataframe"')
 
-    data = read_and_save_data(ASSETS_DIR, 'just_dance_data.csv', _FULL_DATA_URL, to_numpy=False)
-    data['dateComment'] = pd.to_datetime(data['dateComment'])
+    if use_full_size:
+        data = read_and_save_data(ASSETS_DIR, 'just_dance_data.csv', _FULL_DATA_URL, to_numpy=False)
+    else:
+        data = read_and_save_data(ASSETS_DIR, 'just_dance_shorted_data.csv', _SHORT_DATA_URL, to_numpy=False)
+    data[_TIME_COL] = pd.to_datetime(data[_TIME_COL])
+
+    properties = load_properties(as_train_test=False, use_full_size=use_full_size) if include_properties else None
+    embeddings = load_embeddings(as_train_test=False, use_full_size=use_full_size) if include_embeddings else None
 
     if not as_train_test:
-        if not use_full_size:
-            data = data[(data['dateComment'] < '2013-01-01') | (data['dateComment'] >= '2021-01-01')]
         if data_format.lower() != 'textdata':
             return data
 
         label = data.drop(columns=[_TEXT_COL] + _METADATA_COLS).to_numpy().astype(int)
         dataset = TextData(data[_TEXT_COL], label=label, task_type='text_classification',
-                           metadata=data[_METADATA_COLS], categorical_metadata=_CAT_METADATA)
+                           metadata=data[_METADATA_COLS], categorical_metadata=_CAT_METADATA,
+                           properties=properties, categorical_properties=_CAT_PROPERTIES, embeddings=embeddings)
         return dataset
 
     else:
-        if use_full_size:
-            train = data[data['dateComment'] < '2015-01-01']
-            test = data[data['dateComment'] >= '2015-01-01']
-        else:
-            train = data[data['dateComment'] < '2013-01-01']
-            test = data[data['dateComment'] >= '2021-01-01']
+        train = data[data[_TIME_COL] < _DATE_TO_SPLIT_BY]
```

Comment:
can't we use _get_train_test_indexes here as well?

Reply:
we can, will update